components/suggestion/switch_channel_provider.jsx

```diff
@@ -524,4 +524,42 @@ export default class SwitchChannelProvider extends Provider {
             component: ConnectedSwitchChannelSuggestion,
         });
     }
+
+    formatAllRecentChannelsAndDispatch(resultsCallback) {
+        const recentChannels = getAllRecentChannels(getState());
+
+        const channels = [];
+        for (let i = 0; i < recentChannels.length; i++) {
```

Comment:
This can be simplified to:

```suggestion
        for (const channel of recentChannels) {
```

Reply:
applied this change